login-workflow/docs/customization.md

```diff
@@ -0,0 +1,86 @@
+# Customization Guide
+While the default workflow is designed to work for most applications, we understand that you may need to customize the workflow to meet your needs. This guide will walk you through the various ways you can customize the workflow.
+
+## Customizing Registration Workflow
+
+### Customizing the Screen Order
+You may want to customize the order of your screens within the workflow. In order to do this you will need to import the screens you want to use and pass them as children to the `RegistrationWorkflow` component. The screens will render in the order they are passed in.
+
+Our default implementation looks like this:
+  
+```jsx
+<RegistrationWorkflow initialScreenIndex={0}>
+    <EulaScreen />              // screen1
+    <CreateAccountScreen />     // screen2
+    <VerifyCodeScreen />        // screen3
+    <CreatePasswordScreen />    // screen4
+    <AccountDetailsScreen />    // screen5
+</RegistrationWorkflow>
+```
+
+### Custom Screens
+You may inject your own custom screens into the workflow by creating your own custom screens. In order to make this easier for you we have provided components to keep the look and feel consistent with the rest of the workflow. For more information on how to use these components, refer to our [WorkflowCard](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow/components/workflow-card.md) component documentation. After you have created your custom screens, you can pass them as children to the `RegistrationWorkflow` component. The screens will render in the order they are passed in.
+
+Your custom implementation, removing the EulaScreen and VerifyCodeScreen, and adding a custom screen, might look like this:
+
+```jsx
+<RegistrationWorkflow initialScreenIndex={0}>
+    <CreateAccountScreen />    // screen1
+    <MyCustomScreen />         // screen2
+    <CreatePasswordScreen />   // screen3
+    <AccountDetailsScreen />   // screen4
+</RegistrationWorkflow>
+```
+
+<!-- ### Customizing the Registration Success Screen -->
+<!-- @TODO... or not to do. This is just a prop on the screens it is used for. Do we need an in depth example or do the API docs cover it? -->
```

Comment:
I believe it should be a part of RegistrationWorkflow component doc. May be add a line about that and a link to registration workflow component doc. https://github.com/etn-ccis/blui-react-workflows/blob/6ebfced227886744333deac6c4bd4c29f0ecfa3e/login-workflow/docs/components/registration-workflow.md?plain=1#L1

We need to tell users:
- How to add custom success screen to workflow.
- What is the default success screen look (may need to add link to success screen doc). We may need to guide success screen needs organization name and email in the success message, if user is not using organization name and email address in their workflow, they must need to pass custom success screen.
- We shouldn't pass success screen as child.
- An example for custom success screen.